src/format/LemLibOdomGeneratorFormatV0_4.tsx

```diff
@@ -0,0 +1,267 @@
+    minLimit: { value: 0, label: "" },
+    maxLimit: { value: 0, label: "" },
+    step: 0,
+    from: 0,
+    to: 0
+  };
+  @ValidateNumberRange(-Infinity, Infinity)
+  @Expose()
+  bentRateApplicableRange: NumberRange = {
+    minLimit: { value: 0, label: "" },
+    maxLimit: { value: 0, label: "" },
+    step: 0,
+    from: 0,
+    to: 0
+  };
+  @ValidateNumber(num => num >= 0.1 && num <= 255)
+  @Expose()
+  maxDecelerationRate: number = 127;
+
+  @Exclude()
+  readonly format: LemLibOdomGeneratorFormatV0_1;
+
+  @Exclude()
+  public path!: Path;
+
+  constructor(format: LemLibOdomGeneratorFormatV0_1) {
+    this.format = format;
+    makeAutoObservable(this);
+  }
+
+  getConfigPanel() {
+    return <></>;
+  }
+}
+
+// observable class
+export class LemLibOdomGeneratorFormatV0_1 implements Format {
+  isInit: boolean = false;
+  uid: string;
+
+  private gc = new GeneralConfigImpl(this);
+  private readonly events = new Map<keyof HistoryEventMap<CancellableCommand>, Set<Function>>();
+
+  constructor() {
+    this.uid = makeId(10);
+    makeAutoObservable(this);
+  }
+
+  createNewInstance(): Format {
+    return new LemLibOdomGeneratorFormatV0_1();
+  }
+
+  getName(): string {
+    return "LemLib Odometry Code Generator v0.4.x (inch coordinates)";
+  }
+
+  init(): void {
+    if (this.isInit) return;
+    this.isInit = true;
+  }
+
+  getGeneralConfig(): GeneralConfig {
+    return this.gc;
+  }
+
+  createPath(...segments: Segment[]): Path {
+    return new Path(new PathConfigImpl(this), ...segments);
+  }
+
+  getPathPoints(path: Path): PointCalculationResult {
+    const result = getPathPoints(path, new Quantity(this.gc.pointDensity, this.gc.uol));
+    return result;
+  }
+
+  recoverPathFileData(_: string): PathFileData {
+    throw new Error("Loading paths is not supported in this format");
+  }
+
+  exportPathFile(): string {
+    const { app } = getAppStores();
+
+    let rtn = "";
+    const gc = app.gc as GeneralConfigImpl;
+
+    const path = app.interestedPath();
+    if (path === undefined) throw new Error("No path to export");
+    if (path.segments.length === 0) throw new Error("No segment to export");
+
+    const uc = new UnitConverter(this.gc.uol, UnitOfLength.Inch);
+    const points = getDiscretePoints(path);
+
+    if (points.length > 0) {
+      const start = points[0];
+      let heading = 0;
+
+      if (start.heading !== undefined) {
+        heading = fromDegreeToRadian(start.heading);
+      }
+
+      // ALGO: Offsets to convert the absolute coordinates to the relative coordinates LemLib uses
+      const offsets = new Vector(start.x, start.y);
+      for (const point of points) {
+        // ALGO: Only coordinate points are supported in LemLibOdom format
+        const relative = euclideanRotation(heading, point.subtract(offsets));
+        rtn += `${gc.chassisName}.moveTo(${uc.fromAtoB(relative.x).toUser()}, ${uc.fromAtoB(relative.y).toUser()}, ${gc.movementTimeout});\n`;
+      }
+    }
```

Comment:
How are you going to recover the path file if `#PATH.JERRYIO-DATA` line is not included? Or do you think it is not needed?

Reply:
Need to change onClipboardCopy to be specific to this format so we can add the `#PATH.JERRYIO-DATA` section without including it in the clipboard code.

Reply:
The `#PATH.JERRYIO-DATA` section should be added as discussed on Discord, see the next change request.